web/v1/snowflake/CHANGELOG

```diff
@@ -1,4 +1,3 @@
 Version 1.0.0 (2021-02-11)
 --------------------------
-Fix bug in credentials handling for pr_check and e2e scripts (#35)
```

Comment:
Why is this PR touching the snowflake changelog?

Reply:
Because we introduce the CHANGELOG in the root of the repo, which is where we should record issues/commits that aren't specific to a given model. So we're moving the record of all those commits to the top changelog. Same for Redshift.

This does feel a bit of an awkward way to go about it, happy to change approach of course.

Reply:
I feel like we perhaps have diuscussed this before but does the root changelog contain ALL changes/releases? I think I'd be surprised it I opened it and only say "central" fixes in there.
I'm fine that the project specific change logs only include changes related to that release of that subproject though.

Reply:
We did discuss briefly but I don't think we settled on this detail. 

Makes sense though, I'll change it - I was never v certain on this call. :)